kern/common.h

```diff
@@ -23,11 +23,13 @@
 #define AF_INET 2
 #define AF_INET6 10
 #define SA_DATA_LEN 14
+#define ERRNO_DEFAULT 128
```

Comment:
why is it 128 ? why not 0 ?

Reply:
cause [0-128] has practical meaning,0 represents the command exec successfully